taskana-adapter-tasklistener-example/pom.xml

```diff
@@ -1,259 +1,194 @@
-                    </tasks>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
-
-    <profiles>
-        <profile>
-            <id>default</id>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-            </activation>
-            <build>
+                <repository>
+                        <id>jboss-public-repository</id>
+                        <name>JBoss Repository</name>
+                        <url>http://repository.jboss.org/nexus/content/groups/public</url>
+                </repository>
+        </repositories>
+
+        <build>
+                <finalName>${project.artifactId}</finalName>
                 <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-surefire-plugin</artifactId>
-                    </plugin>
+                        <plugin>
+                                <!-- Deploy to JBoss AS7: mvn clean jboss-as:deploy See also: https://docs.jboss.org/jbossas/7/plugins/maven/latest/examples/deployment-example.html -->
+                                <groupId>org.jboss.as.plugins</groupId>
+                                <artifactId>jboss-as-maven-plugin</artifactId>
+                                <version>${version.jboss-as-maven-plugin}</version>
+                        </plugin>
+                        <plugin>
+                                <!-- Deploy to Wildfly: mvn clean wildfly:deploy See also: https://docs.jboss.org/wildfly/plugins/maven/latest/examples/deployment-example.html -->
+                                <groupId>org.wildfly.plugins</groupId>
+                                <artifactId>wildfly-maven-plugin</artifactId>
+                                <version>${version.wildfly.maven.plugin}</version>
+                        </plugin>
+                        <plugin>
+                                <!-- Deploy to JBoss using: mvn clean package antrun:run
+                     Follow the instructions in build.properties.example to make it work!-->
+                                <groupId>org.apache.maven.plugins</groupId>
+                                <artifactId>maven-antrun-plugin</artifactId>
+                                <configuration>
+                                        <tasks>
+                                                <ant antfile="${basedir}/build.xml">
+                                                        <target name="copy.war.into.jboss" />
+                                                </ant>
+                                        </tasks>
+                                </configuration>
+                        </plugin>
                 </plugins>
-            </build>
-        </profile>
-
-    </profiles>
-
-
-</project>
\ No newline at end of file
+        </build>
+
+        <profiles>
+                <profile>
+                        <id>default</id>
```

Comment:
if this profile is active by default, why not move the surefire-plugin into <build> ?

Reply:
We don't have unit tests for this example and don't plan to create them.